api/src/org/labkey/api/audit/AuditHandler.java

```diff
@@ -64,12 +64,15 @@ static Pair<Map<String, Object>, Map<String, Object>> getOldAndNewRecordForMerge
         for (Map.Entry<String, Object> entry : existingRow.entrySet())
         {
             String key = entry.getKey();
-            String nameFromAlias = columns
-                    .stream()
-                    .filter(column -> column.getAlias().equalsIgnoreCase(key))
-                    .map((ColumnInfo::getName))
-                    .findFirst()
-                    .orElse(key);
+            // getDatasetRows() (at least) should returns key==column.getName(), expect getColumn(name) to work
```

Comment:
```suggestion
            // getDatasetRows() (at least) should return key==column.getName(), expect getColumn(name) to work
```

(at least) compared with what alternative?